src/js/main.js

```diff
@@ -129,43 +129,48 @@ function startMaqueeMarq() {
 const mouse = {
 	x: 0,
 	y: 0,
-	endX: 0,
-	endY: 0,
 	dragCallback: false, // What to do when a dragged element is moved
-	endCallback: false // What to do when a dragging stops
+	endCallback: false, // What to do when a dragging stops
+	executeCallback: function(e) {
+		if (mouse.dragCallback) {
+			if (e.cancelable) {
+				e.preventDefault();
+			}
+			mouse.dragCallback(e);
+		}
+	}
 };
-
-window.onmousemove = e => {
+window.addEventListener("mousemove", e => {
 	mouse.x = e.clientX;
 	mouse.y = e.clientY;
-	if (mouse.dragCallback) {
-		e.preventDefault();
-		mouse.dragCallback(e);
-	}
-};
-window.onmouseup = () => {
+	mouse.executeCallback(e);
+});
+window.addEventListener("touchstart", e => {
+	mouse.x = e.changedTouches[0].clientX;
```

Comment:
There is no support for this in Safari, according to https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/changedTouches... Would that be a problem?

Reply:
Don't think so unless there are touch devices supporting Safari on OSX?

Reply:
I guess not! 👍